src/screens/Profile/Profile.jsx

```diff
@@ -134,35 +137,24 @@ const Profile = () => {
       >
         <h1 style={{ margin: "auto" }}>Hello World</h1>
       </Modal>
-      <Modal
+
+      <ConfirmDialog
         title={t("screens.profile.modals.deleteAccount.heading")}
-        open={showDeleteAccountModal}
+        onSubmit={onDelete}
         onClose={closeDeleteAccountModal}
```

Comment:
can you also clear the `deleteConfirmation` value in the `closeDeleteAccountModal` function?